pkg/identity/identity_naming_standard.go

```diff
@@ -53,10 +47,15 @@ func (s *identityNamingStandard) IdentityName() string {
 	return fmt.Sprintf("%s:%s", s.provider, s.username())
 }
 
-// isIdentityNameCompliant returns true if the specified name is RFC-1123 compliant, otherwise it returns false
+// isIdentityNameCompliant returns true if the specified name is compliant with the Openshift identity naming standard,
+// encapsulated in the code found in the urlEncodeIfNecessary() function, otherwise it returns false.
+//
+// The code at time of writing can be found at:
+//
+// https://github.com/openshift/oauth-server/blob/master/pkg/api/types.go#L108
```

Comment:
I would use a specific commit rather than master in case master is updated in the future:
```suggestion
// https://github.com/openshift/oauth-server/blob/ef385cc3c9d90ee52f6db211ceb751e04ae967f5/pkg/api/types.go#L108
```